metricbeat/module/apache/status/data.go

```diff
@@ -141,6 +163,9 @@ func eventMapping(scanner *bufio.Scanner, hostname string) (common.MapStr, *s.Er
 		},
 	}
 	_, err := schema.ApplyTo(event, fullEvent)
+	if err != nil {
```

Comment:
Is there a way to identify on the first request which Apache version it is to then pick the schema? Otherwise we decode every event twice.

Reply:
Umm, indeed I can do it depend on the different fields seen in `fullEvent` instead of completelly decoding it twice.